digdag-standards/src/main/java/io/digdag/standards/operator/pg/PgConnection.java

```diff
@@ -116,7 +119,7 @@ protected StatusRow lockStatusRow(UUID queryId)
         {
             try (Statement stmt = connection.createStatement()) {
                 ResultSet rs = stmt.executeQuery(String.format(ENGLISH,
-                            "SELECT completed_at FROM %s WHERE query_id = '%s' FOR UPDATE",
+                            "SELECT completed_at FROM %s WHERE query_id = '%s' FOR UPDATE NOWAIT",
```

Comment:
👍 
I understand that the retry counter in Digdag will be incremented if the original exception is thrown as is. So we need to take care of that error.